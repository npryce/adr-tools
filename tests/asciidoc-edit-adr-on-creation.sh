echo "adoc" > .adr-file-type
export EDITOR
export VISUAL

EDITOR=fake-editor
VISUAL=
adr new created with EDITOR environment variable set

EDITOR=
VISUAL=fake-visual
adr new created with VISUAL environment variable set

EDITOR=fake-editor
VISUAL=fake-visual
adr new uses setting of VISUAL if both VISUAL and EDITOR are set
