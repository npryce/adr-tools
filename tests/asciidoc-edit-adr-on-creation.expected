echo "adoc" > .adr-file-type
export EDITOR
export VISUAL

EDITOR=fake-editor
VISUAL=
adr new created with EDITOR environment variable set
EDITOR doc/adr/0001-created-with-editor-environment-variable-set.adoc
doc/adr/0001-created-with-editor-environment-variable-set.adoc

EDITOR=
VISUAL=fake-visual
adr new created with VISUAL environment variable set
VISUAL doc/adr/0002-created-with-visual-environment-variable-set.adoc
doc/adr/0002-created-with-visual-environment-variable-set.adoc

EDITOR=fake-editor
VISUAL=fake-visual
adr new uses setting of VISUAL if both VISUAL and EDITOR are set
VISUAL doc/adr/0003-uses-setting-of-visual-if-both-visual-and-editor-are-set.adoc
doc/adr/0003-uses-setting-of-visual-if-both-visual-and-editor-are-set.adoc
